diff --git a/etc/builder.py b/etc/builder.py
index 4aeea51..59981ec 100644
--- a/etc/builder.py
+++ b/etc/builder.py
@@ -1,3 +1,5 @@
+from __future__ import print_function
+
 from iocbuilder import Device, AutoSubstitution, Architecture, SetSimulation
 from iocbuilder.arginfo import *
 from iocbuilder.modules.asyn import Asyn, AsynPort, AsynIP
@@ -32,7 +34,7 @@ class SimHDF5Detector(AsynPort):
     DbdFileList = ['simHDF5Support']
 
     def Initialise(self):
-        print '# SimHDF5DetectorConfig(portName, maxBuffers, maxMemory )'
-        print 'SimHDF5DetectorConfig( %(PORT)10s, 0, %(MEMORY)9d )' % self.__dict__
+        print('# SimHDF5DetectorConfig(portName, maxBuffers, maxMemory )')
+        print('SimHDF5DetectorConfig( %(PORT)10s, 0, %(MEMORY)9d )' % self.__dict__)
 
 
