diff --git a/prepareCustom.sh b/prepareCustom.sh
index 0dd21e9..f92e4e0 100755
--- a/prepareCustom.sh
+++ b/prepareCustom.sh
@@ -50,7 +50,5 @@ sed -e '7i export hostname=$(hostname -s)\' \
 sed -e "s/.*.db'/&, hostname=\$(hostname)/" \
     -i $1/iocBoot/ioc${2}/st${2}.src
 sed -e "s/${2}Top.edl/-m hostname=\$(hostname -s) ${2}Top.edl/" \
-    -e '2i export EPICS_CA_SERVER_PORT=6064' \
-    -e '3i export EPICS_CA_REPEATER_PORT=6065' \
     -i $1/${2}App/opi/edl/st${2}-gui
 
