diff --git a/malcolm/modules/builtin/defines.py b/malcolm/modules/builtin/defines.py
index ed09ac5a..e9dc3838 100644
--- a/malcolm/modules/builtin/defines.py
+++ b/malcolm/modules/builtin/defines.py
@@ -92,7 +92,7 @@ def cmd_string(name, cmd):
     # type: (AName, ACmd) -> ADefine
     """Define a string parameter coming from a shell command to be used within
     this YAML file. Trailing newlines will be stripped."""
-    value = subprocess.check_output(cmd, shell=True).rstrip("\n")
+    value = subprocess.check_output(cmd, shell=True).decode().rstrip("\n")
     return Define(name, value)

