diff --git a/makeSupport/Makefile b/makeSupport/Makefile
index 8117a0d3..a62d4ec6 100644
--- a/makeSupport/Makefile
+++ b/makeSupport/Makefile
@@ -43,6 +43,6 @@ TEMPLATES += streamSCPI/_NAME_Sup/dev_NAME_.proto
 TEMPLATES += streamSCPI/_NAME_Sup/dev_NAME_.db
 TEMPLATES += streamSCPI/documentation/dev_NAME_.html
 
-SCRIPTS_HOST += makeSupport.pl
+SCRIPTS_HOST += ../makeSupport.pl
 
 include $(TOP)/configure/RULES
