diff --git a/iocbuilder/support.py b/iocbuilder/support.py
index aacfc0b..6c7310c 100644
--- a/iocbuilder/support.py
+++ b/iocbuilder/support.py
@@ -1,5 +1,6 @@
 '''A miscellaneous collection of fairly generic utilities.'''
 import fnmatch
+import inspect
 import itertools
 import os
 import os.path
@@ -21,7 +22,8 @@ __all__ = ['Singleton', 'AutoRegisterClass', 'SameDirFile', 'quote_c_string']
 def ExportModules(globals, *modulenames):
     names = []
     for modulename in modulenames:
-        module = import_module(f"iocbuilder.{modulename}")
+        caller_pname = inspect.currentframe().f_back.f_globals['__name__']
+        module = import_module(f"{caller_pname}.{modulename}")
         if hasattr(module, '__all__'):
             for name in module.__all__:
                 globals[name] = getattr(module, name)
